src/Symfony/Component/Form/Extension/Core/DataTransformer/WeekToArrayTransformer.php

```diff
@@ -26,7 +26,7 @@ class WeekToArrayTransformer implements DataTransformerInterface
      *
      * @param string|null $value A week date string
      *
-     * @return array A value containing year and week
+     * @return array{year: int, week: int}
```

Comment:
```suggestion
     * @return array{year: int|null, week: int|null}
```

See line 37 (the Psalm comment)